configure.ac

```diff
@@ -83,10 +83,10 @@ AC_ARG_ENABLE(build32,
         [CFLAGS="$CFLAGS";CXXFLAGS="$CXXFLAGS";LIBALIC_FLAG="";LWIP_FLAG=""])
 
 # gcov support
-AC_ARG_ENABLE(gcov, AC_HELP_STRING([--enable-gcov], [turn on code coverage analysis tools]),[CFLAGS+="--coverage -O0";CPPFLAGS+="--coverage -O0";LIBS+="-lgcov";VMA_GCOV=1],[VMA_GCOV=0])
+AC_ARG_ENABLE(gcov, AC_HELP_STRING([--enable-gcov], [turn on code coverage analysis tools]),[CFLAGS+=" --coverage -O0";CPPFLAGS+=" --coverage -O0";LIBS+=" -lgcov";VMA_GCOV=1],[VMA_GCOV=0])
 
 # time_measure support
-AC_ARG_ENABLE(time_measure, AC_HELP_STRING([--enable-time_measure], [turn on time measuring]),[CFLAGS+="-DVMA_TIME_MEASURE";CPPFLAGS+="-DVMA_TIME_MEASURE";],[])
+AC_ARG_ENABLE(time_measure, AC_HELP_STRING([--enable-time_measure], [turn on time measuring]),[CFLAGS+=" -DVMA_TIME_MEASURE";CPPFLAGS+=" -DVMA_TIME_MEASURE";],[])
```

Comment:
I think it could be done too if you says about ` --coverage -O0` but I prefer to set compiler specific options separately (see https://github.com/Mellanox/libvma/blob/master/configure.ac#L228).
CFLAGS should be used for CC and CXXFLAGS for CXX. CPPFLAGS can be used for common options as -I or -D etc.